src/target.jl

```diff
@@ -3,6 +3,15 @@
 
 #using Distributions
 
+struct TESSTarget
```

Comment:
I like having separate structs for TESSTarget and KeplerTarget.  I'd suggest adding
`abstract struct TargetAbstract end
`
then we can make both  TESSTarget and KeplerTarget subtypes of AbstractTarget like
`struct TESSTarget <: TargetAbstract
`
so as to reduce the need for duplicated code.  E.g., for many functions that use member variables in common, we can do something like
`num_planets(t::T) where {T<:TargetAbstract} = sum( num_planets, t.sys)
`